packages/fxa-auth-server/test/local/push.js

```diff
@@ -2,1025 +2,802 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use strict';
+"use strict";
```

Comment:
There is a lot of churn in this file, so much that github is hiding it by default. Summary:

* I added two new tests for the success/failure log info and statsd calls.
* I asyncified the existing tests, which were using explicit promise chains.
* I migrated the exists tests to use higher-level `sinon.assert` functions for (IMHO) improved clarity.
* I updated the assertions in existing tests where appropriate, to match the logging changes.